jenkinsfiles/ios_jenkinsfile.groovy

```diff
@@ -84,4 +92,4 @@ node('ios') {
     stage('Archive') {
         archiveArtifacts "build/Debug-iphoneos/${OUTPUT_FILE_NAME}"
```

Comment:
you can send an additional argument in xcodeBuild function: `config: 'Release'` or `config: 'Debug'` - default is "Debug" if no value is provided.